packages/tasks/src/components/RecommendedRibbon.js

```diff
@@ -2,20 +2,19 @@
  * External dependencies
  */
 import { __ } from '@wordpress/i18n';
+import { Pill } from '@woocommerce/components';
 
 const localPartners = [ 'mercadopago' ];
 
 export const RecommendedRibbon = ( { methodKey, isPill = false } ) => {
-	const classes = isPill
-		? 'woocommerce-task-payment__recommended-pill'
-		: 'woocommerce-task-payment__recommended-ribbon';
-
 	const text = localPartners.includes( methodKey )
 		? __( 'Local Partner', 'woocommerce-admin' )
 		: __( 'Recommended', 'woocommerce-admin' );
 
-	return (
-		<div className={ classes }>
+	return isPill ? (
```

Comment:
Any reason why we're not starting out with `ts` on new components? It would really help if we could start with typescript rather than relying on refactoring to get to it later.

Reply:
> Maybe we can simplify this to:

Thanks @joshuatf , I actually didn't like having a `<Pill />` returned from a component called `<RecommendationRibbon>`, so I ended up factoring this out of it. 

> Any reason why we're not starting out with ts on new components? 

@samueljseay Really just due to the fact that it would take me twice as long at this stage, and we're already having to bring in members from other teams to try to get this over the finish line this sprint. If we decide as a team to prioritize TS over speed in the short/medium term, then I'm all for it. Otherwise I'm just approaching it as time allows or my own stubbornness insists.

Reply:
@joelclimbsthings totally fair enough :) I appreciate you considering the pragmatic path forward 👍